src/demandlib/bdew/elec_slp.py

```diff
@@ -204,3 +204,21 @@ def get_profile(self, ann_el_demand_per_sector):
             ).dropna(how="all", axis=1)
             * 4
         )
+
+    def get_profiles(self, *args):
+        if len(args) == 0:
+            return self.slp_frame
+        else:
+            return self.slp_frame[list(args)]
+
+    def scale_profiles(self, ann_el_demand_per_sector):
```

Comment:
To me, `scale_profiles` sounds like in place scaling, so that later calls to `get_profiles` will return scaled values. I suggest `get_scaled_profiles`.